//
// PROJECT:         Aspia Remote Desktop
// FILE:            proto/host_user.proto
// LICENSE:         See top-level directory
// PROGRAMMERS:     Dmitry Chapyshev (dmitry@aspia.ru)
//

syntax = "proto3";

option optimize_for = LITE_RUNTIME;

package aspia.proto;

message HostUser
{
    bool enabled         = 1;
    string username      = 2;
    bytes password_hash  = 3;
    uint32 session_types = 4;
}

message HostUserList
{
    repeated HostUser user_list = 1;
}
